frontend/src/main/scala/bloop/engine/caches/LastSuccessfulResult.scala

```diff
@@ -11,21 +11,18 @@ import bloop.task.Task
 
 import monix.execution.atomic.AtomicInt
 import xsbti.compile.CompileAnalysis
-import xsbti.compile.FileHash
 import xsbti.compile.MiniSetup
 import xsbti.compile.PreviousResult
 
 case class LastSuccessfulResult(
-    sources: Vector[UniqueCompileInputs.HashedSource],
```

Comment:
This was unused, so no need to keep in memory especially that we now have also file contents